lib/engine/game/allowed_company_purchase_prize_between_50_and_150_percent.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+#
+# This module sets up so that the allowed price
+# for purchasing private companies are between
+# 50% and 150% of the face value.
+# This is used in e.g 18MEX and 18AL.
+#
+module AllowedCompanyPurchasePrizeBetween50And150Percent
```

Comment:
the name is too long

Reply:
Lol, yes a bit long. Would **CompanyPrize50To150Percent** do?